test/test_torch.py

```diff
@@ -542,6 +542,36 @@ def test_multidim(x, dim):
     def test_dim_reduction(self):
         self._test_dim_reduction(self, lambda t: t)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_cpu_parallel(self):
+        # To use parallel branches we'll need to compare on tensors
+        # that are relatively large. Even if this is run on a single
+        # core machine these tests will still give you signal on
+        # the correctness
+
+        def _run_test(size):
+            for dim in range(len(size) + 1):
+                nv = np.round(np.random.rand(*size))  # 0s and 1s
```

Comment:
Can’t we just use .bernoulli_() and drop the numpy dependency?

Reply:
No, since we're using numpy's sum as well to compare with torch's sum.

Reply:
Couldn't we use `.tolist()`, flatten the lists and use the `sum()` Python builtin?